test/unit/spec/narrative_core/jobCommChannel-spec.js

```diff
@@ -167,55 +113,51 @@ define(['jobCommChannel', 'base/js/namespace', 'common/runtime'], (
             ['request-job-info', { jobId: 'someJob', parentJobId: 'someParent' }],
         ];
         busMsgCases.forEach((testCase) => {
-            it('Should handle ' + testCase[0] + ' bus message', (done) => {
+            it('Should handle ' + testCase[0] + ' bus message', () => {
```

Comment:
convert this arrow function to `function() {` to avoid messing with Jasmine's `this`